cmd/minikube/cmd/start.go

```diff
@@ -81,19 +82,37 @@ func runStart(cmd *cobra.Command, args []string) {
 	}
 
 	var host *host.Host
-	start := func() (err error) {
-		host, err = cluster.StartHost(api, config)
-		if err != nil {
-			glog.Errorf("Error starting host: %s. Retrying.\n", err)
-		}
-		return err
-	}
-	err := util.Retry(3, start)
+
+	exists, err := cluster.HostExists(api)
 	if err != nil {
-		glog.Errorln("Error starting host: ", err)
+		glog.Errorf("Error checking if host exists: %s", err)
 		cmdUtil.MaybeReportErrorAndExit(err)
 	}
 
+	if !exists {
+		fmt.Println("Creating new minikube VM ...")
+		host, err = cluster.CreateHost(api, machineConfig)
+		if err != nil {
+			glog.Errorf("Error creating host: %v", err)
+			cmdUtil.MaybeReportErrorAndExit(err)
+		}
+		// Save the machine settings for the new machine
+		config.WriteConfig(viper.AllSettings())
+	} else {
```

Comment:
libmachine api.Create both creates and starts.  So I'm not sure we can separate the actions downstream.  (Although our cluster.Start is _mostly_ noop if the machine is already running)